assemblyline/odm/models/ontology/ontology.py

```diff
@@ -12,11 +12,12 @@
 from assemblyline.odm.models.ontology.file import File
 
 Classification = forge.get_classification()
-ODM_VERSION = "1.10"
+ODM_VERSION = "1.11"
 
 
 @odm.model(index=False, store=False, description="Heuristics raised")
 class Heuristics(odm.Model):
+    attack_ids = odm.List(odm.Keyword(), description="ATT&CK IDs associated to heuristic")
```

Comment:
I think we should reflect the whole Attack structure that is stored in odm.model.result.